entity-framework/core/what-is-new/ef-core-7.0/whatsnew.md

```diff
@@ -466,22 +469,24 @@ WHERE CAST(JSON_VALUE([p].[Metadata],'$.Views') AS int) > 3000
 -->
 [!code-csharp[UpdateDocument](../../../../samples/core/Miscellaneous/NewInEFCore7/JsonColumnsSample.cs?name=UpdateDocument)]
 
-In this case, the entire new document is passed as a parameter to the `Update` command:
+In this case, the entire new document is passed as a parameter:
 
 ```text
 info: 8/30/2022 20:21:24.392 RelationalEventId.CommandExecuted[20101] (Microsoft.EntityFrameworkCore.Database.Command)
       Executed DbCommand (2ms) [Parameters=[@p0='{"Phone":"01632 88346","Address":{"City":"Trimbridge","Country":"UK","Postcode":"TB1 5ZS","Street":"2 Riverside"}}' (Nullable = false) (Size = 114), @p1='2'], CommandType='Text', CommandTimeout='30']
 ```
 
+Which is then used in the `UPDATE` SQL:
+
 ```sql
-      SET IMPLICIT_TRANSACTIONS OFF;
-      SET NOCOUNT ON;
-      UPDATE [Authors] SET [Contact] = @p0
-      OUTPUT 1
-      WHERE [Id] = @p1;
+SET IMPLICIT_TRANSACTIONS OFF;
+SET NOCOUNT ON;
+UPDATE [Authors] SET [Contact] = @p0
```

Comment:
I'd just strip the two SET statements as they're not relevant or interesting. Also below.